packages/block-editor/src/components/block-list-appender/index.js

```diff
@@ -30,19 +31,32 @@ function BlockListAppender( {
 	rootClientId,
 	canInsertDefaultBlock,
 	isLocked,
-	renderAppender: CustomAppender,
+	renderAppender: CustomAppender, // Deprecated.
+	appender,
 	className,
 	selectedBlockClientId,
 	tagName: TagName = 'div',
 } ) {
-	if ( isLocked || CustomAppender === false ) {
+	if ( undefined !== CustomAppender ) {
+		deprecated( 'renderAppender prop', {
+			since: '10.9',
+			alternative: 'appender prop',
+		} );
+	}
+
+	if (
+		isLocked ||
+		( undefined !== appender && ! appender ) ||
```

Comment:
It's not undefined, but it is falsey, so I imagine this is to check if it's `null` or `false`? e.g. values that people often use to render nothing in React.

Maybe it could be `( appender === null || appender === false )` to be more explicit.

Reply:
Sure! It makes more sense! https://github.com/WordPress/gutenberg/pull/29911/commits/25317c9352acca0c96e7ef004ef721616a5f8bda